exercises/practice/resistor-color/.meta/config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "authors": [],
```

Comment:
Username is missing here.

Reply:
No anonymous exercises?

Corrected! Thanks for the catch.